src/example/components/text-only.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component, PropTypes} from 'react';
+import find from 'lodash/find';
+import {connect as connectToDocumentation} from '../../behaviours/documentation';
+import {connect as connectToForm } from '../../behaviours/form';
+import {connect as connectToMetadata} from '../../behaviours/metadata';
+import {connect as connectToFieldHelpers} from '../../behaviours/field';
+import {connect as connectToMasterData} from '../../behaviours/master-data';
+import {loadUserAction, saveUserAction} from '../actions/user-actions';
+
+import Panel from '../../components/panel';
+import compose from 'lodash/flowRight';
+
+class UserTextOnly extends Component {
+    componentWillMount() {
+        const {id, load, loadMasterData} = this.props;
+        load({id});
+        loadMasterData();
+    }
+    render() {
+        const {textFor} = this.props;
+        return (
+            <Panel title='Text only user page' {...this.props}>
+                <div>{textFor('uuid', {entityPath: 'user.information'})}</div>
```

Comment:
Maybe an example where `entityPath` is not write because this works. `{textFor('firstName')}`